integration-tests/README.md

```diff
@@ -0,0 +1,107 @@
+ - Install the latest version of WebLogic Deploy Tooling (WDT) and WebLogic Image Tool (WIT)
+ - Install supported istio service mesh.
+ - Pull the specified version of WebLogic image from container-registry.oracle.com (OCR)
+ - Install WebLogic Kubernetes Operator.
+ - Build a WebLogic domain resource.
+ - Make sure WebLogic domain is running by verifying the corresponding server pod and kubernetes service status. 
+ - After test execution, delete all the Namespaces, Kubernetes Objects created during test execution.
+ - Archive the test standard output for each test class and kubernetes object details for each test method in the diagnostic directory for triage when tests fail or when the env variable COLLECT_LOGS_ON_SUCCESS is set to true.
+ 
+# How to run Operator integration tests locally on Oracle Linux
+
+- Install supported version of Helm and Kubernetes cluster. For detail see [Weblogic Kubernetes Operator guide](https://oracle.github.io/weblogic-kubernetes-operator/userguide/introduction/introduction/).
+- Install supported version of JDK. Need JDK Version 11+ and above.
+- Clone the weblogic-kubernetes-operator repository
+- Setup a personal account on container-registry.oracle.com
+- Log into https://container-registry.oracle.com and accept Oracle Standard Terms and Restrictions to pull WebLogic images from container-registry.oracle.com/middleware/weblogic
+- Export the following environment variable before running the tests
+   ```
+    export BASE_IMAGES_REPO="container-registry.oracle.com"
+    export OCR_EMAIL=<ocr_email>
+    export OCR_USERNAME=<ocr_username>
+    export OCR_PASSWORD=<ocr_password> 
+   ```
+- `cd  weblogic-kubernetes-operator` and run the following commands
+  ```
+    # Build the Weblogic Kubernetes Operator binary 
+    mvn clean install 
+    # Run tests in sequential
+    mvn -pl -integration-tests -P integration-tests clean verify
+    # Run tests in parallel
+    mvn -DPARALLEL=true -DNUMBER_OF_THREADS=2 -pl -integration-tests -P integration-tests verify
+   ```
+## How to run individual test class(es)
+mvn -Dit.test=ItMiiDomain,ItMiiUpdateDomainConfig -pl integration-tests -P integration-tests verify
+
+## How to run a single test method in a test class 
+mvn -Dit.test="ItMiiUpdateDomainConfig#testMiiDeleteSystemResources" -pl integration-tests -P integration-tests verify
+
+## How to exclude test class(es) 
+mvn -Dit.test="!ItCrossDomainTransaction,!ItMiiUpdateDomainConfig" -pl integration-tests -P integration-tests verify
+
+## Environment variables to manage test execution 
+| Variable | Description | Default Value
+| --- | --- | --- |
+| SKIP_CLEANUP  | Test infrastructure removes all kubernetes objects created during test execution. To retain all such objects for triaging test failures this environment variable should be set to true. User need to run weblogic-kubernetes-operator/src/integration-tests/bash/cleanup.sh to clean up kubernetes objects after traiging.    | false
```

Comment:
SKIP_CLEANUP ->`SKIP_CLEANUP` (surround with back ticks to give it code font. Please do this for every variable name!)
kubernetes  -> Kubernetes 
failures -> failures, (comma)
User  -> Users
weblogic-kubernetes-operator/src/integration-tests/bash/cleanup.sh -> `weblogic-kubernetes-operator/src/integration-tests/bash/cleanup.sh` (code font)
kubernetes  -> Kubernetes (please capitalize Kubernetes everywhere in the README)

Reply:
done